bioimage_manifest.yaml

```diff
@@ -0,0 +1,8 @@
+name: Example Models
+description: A repository for example models
+
+models:
+  - UNet2d.model.yaml
+
+transforms:
+  - Normalization.transform.yaml
```

Comment:
currently in use is the somewhat bulky `NormalizeZeroMeanUnitVariance` transformation...